src/epcpm/mainwindow.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>1200</width>
-    <height>900</height>
+    <width>1516</width>
+    <height>983</height>
```

Comment:
Unless there is a reason to change the size, I'd leave these alone.

Reply:
These changes were added automatically by the Qt designer tool. Reverted.